usr/share/rear/output/default/980_umount_output_dir.sh

```diff
@@ -8,4 +8,4 @@ if [[ -z "$OUTPUT_URL" ]] ; then
     return
 fi
 
-umount_url $OUTPUT_URL $BUILD_DIR/outputfs
+umount_url "$OUTPUT_URL" $BUILD_DIR/outputfs
```

Comment:
`$BUILD_DIR/outputfs` should be quoted as well for consistency with other changes

Reply:
I didn't quote everything indeed, because I wanted minimal changes
but I agree this can be quoted as well, even though we know it's not problematic since under control somehow

Reply:
I see, I checked only lines that you touched in any case, to not increase the volume of lines changed.

I don't entirely agree with "under control" though, as users can set BUILD_DIR to something that contains spaces or other special characters (although this is unlikely and fixing it is not a priority).